cors_filter.go

```diff
@@ -37,15 +40,37 @@ func (c CrossOriginResourceSharing) Filter(req *Request, resp *Response, chain *
 		chain.ProcessFilter(req, resp)
 		return
 	}
-	if len(c.AllowedDomains) > 0 { // if provided then origin must be included
-		included := false
-		for _, each := range c.AllowedDomains {
-			if each == origin {
-				included = true
+	if len(c.AllowedDomains) > 0 { // if provided then origin must be allowed
```

Comment:
seeing some duplicate code here and in isOriginAllowed, is it possible to call that method here instead?